src/internal/SwitchBase.js

```diff
@@ -173,7 +172,6 @@ export function createSwitch({
           buttonRef={(c) => { this.button = c; }}
           className={className}
           disabled={disabled}
-          ripple={ripple}
```

Comment:
There are quite a few of these, but many the tests fail if the prop isn't explicitly defined and is only passed in `...other`.